account_accountant_oca/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2020 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Accountant (OCA)",
+    "summary": "Add missing features for Odoo Community Edition"
+    " to configure properly accounting. (Fiscal years, Overdue Message, ...)",
+    "version": "12.0.1.0.1",
+    "category": "Accounting",
+    "author": "GRAP,Odoo Community Association (OCA)",
+    "maintainers": ["legalsylvain"],
+    "website": "http://www.github.com/OCA/account-financial-tools",
+    "license": "AGPL-3",
```

Comment:
If this module is not compatible with account_accountant, you should add this line:

`    'excludes': ['account_accountant'],`

Reply:
For most OCA developers it is not possible to know what is in which enterprise module. So I would prefer that we don't fill this list with enterprise modules as it is not possible to maintain it.